inventories/default/group_vars/all.yaml.template

```
@@ -270,9 +270,17 @@ hypershift:
     mce_namespace: "multicluster-engine" # This is the Recommended Namespace for Multicluster Engine operator
 
   agents_parms:
+    static_ip_parms:
+      static_ip: 'true' 
+      ip:     # Required only if static_ip is true
+      - #
+
+      interface: 'eth0'
+      generate_mac_addr: 'true'
     agents_count: 
-    # Give the list of macaddress for agents here separated by ,
-    agent_mac_addr: [  ]
+    # Give the list of macaddress for agents here if you want to use specific mac addresses
+    agent_mac_addr:  
+    - #
```

```suggestion
    # Give the list of macaddress for agents here if you want to use specific mac addresses
    agent_mac_addr:  
      #- 
```
Instead of a variable defining whether mac addresses are manually specified by the user or randomly generated, it should be determined automatically based on whether the variable is defined or not. What do you think @veera-damisetti ?

Defined -> use that mac address
Undefined -> randomly generate one

---

Could be the same for whether or not static IPs are used too

---

@jacobemery  this sounds good . Will update

---

Updated. 
Thanks ...!